Data/DynamicInputTexture.cs

```diff
@@ -134,13 +141,29 @@ public double ScaleFactor
             }
             set
             {
-                _scale_factor = value;
+                _scale_factor = Smooth(value,2);
                 OnPropertyChanged(nameof(ScaleFactor));
-                Regions.ToList().ForEach(x => x.ScaleFactor = value);
+                Regions.ToList().ForEach(x => x.ScaleFactor = _scale_factor);
             }
         }
         #endregion
 
+        private double Smooth(double value, int accuracy)
+        {
+            double Factor = ((int)value * 10).ToString().Length;
```

Comment:
This is strange (converting to a string), can you explain this?

Also please lowercase the variable since it's not a property.

Reply:
I admit it's a bit strange.
this was the easiest metode for me to get the length.